packages/dx-react-grid/docs/reference/search-panel.md

```diff
@@ -0,0 +1,48 @@
+# SearchingPanel Plugin Reference
+
+A plugin that renders a searching panel.
+
+## User Reference
+
+### Dependencies
+
+- [SearchingState](searching-state.md)
+- [IntegratedFiltering](integrated-filtering.md)
+- [Toolbar](toolbar.md)
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+rootComponent | ComponentType&lt;[SearchPanel.RootProps](#searchpanelrootprops)&gt; | | A component that renders the search panel root element.
+messages? | [SearchPanel.LocalizationMessages](#localization-messages) | | An object that specifies localization messages.
+
+## Interfaces
+
+### SearchPanel.RootProps
+
+Describes properties passed to a component that renders the search panel root element.
+
+Field | Type | Description
+------|------|------------
+children? | ReactNode | A React node to be placed in the search panel.
+getMessage | ([messageKey](#localization-messages): string) => string | Returns the search panel placeholder text. Available in the "@devexpress/dx-react-grid-material-ui" package.
+
+## Localization Messages
+
+Field | Type | Default | Description
+------|------|---------|------------
+searchPlaceholder? | string | 'Search...' | The search editor placeholder text. Available in the "@devexpress/dx-react-grid-material-ui" package.
```

Comment:
Why do we avoid text placeholder in BS3?

Reply:
I have this legacy from other plugins like filter, column chooser, paging panel & etc.